### ISSUE TITLE
Keep getting this error in atom

### ISSUE THREAD

**Issue by ntotten** [auth0/javascript#16]
Error: /Users/ntotten/git/auth0/auth0-vagrant/src/auth0-docs/node_modules/eslint-config-auth0/rules/react-a11y.js: Configuration for rule "jsx-a11y/anchor-has-content" is invalid: Severity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed "off").

Referenced from: auth0 Referenced from: /Users/ntotten/git/auth0/auth0-vagrant/src/auth0-docs/.eslintrc

**Comment by ramasilveyra** [comment]
Atom eslint plugin tries to find the package `eslint-config-auth0` in this way:
1- First on your local root repo folder (aka `/Users/ntotten/git/auth0/auth0-vagrant/src/auth0-docs/node_modules`).
2- Then on your global node modules.
And if doesn't find them it starts to throw errors like that.

If you have installed the required eslint modules only inside vagrant, the atom plugin 
it will not find them so it will throw that error.

A solution can be install the eslint deps locally on `/Users/ntotten/git/auth0/auth0-vagrant/src/auth0-docs/`, or globally (make sure that you use the same node version/path for the atom plugin and the global modules, _careful with tools like nvm_, **locally it's preferable**): 

``` bash
sudo npm i -g eslint@3 eslint-plugin-jsx-a11y@2 eslint-plugin-import@1 eslint-plugin-react@6 eslint-config-auth0@6
```

After that close and re open Atom.

Already solved via slack so I will close the issue :), just for the record.
